packages/reports/tests/unit/consumers/request/sort-test.js

```diff
@@ -50,52 +50,112 @@ test('UPSERT_SORT', function(assert) {
 test('REMOVE_SORT', function(assert) {
   assert.expect(1);
 
-  const removeSortByMetricName = (metric) => {
-    assert.equal(metric,
+  const removeSortByMetricName = (metricName) => {
+    assert.equal(metricName,
       'click',
-      'metricSort is removed from the request');
+      'removeSortByMetricName is called with correct metric name');
   };
 
   let currentModel = { request: {
         removeSortByMetricName
       }},
       consumer = this.subject();
 
-  consumer.send(RequestActions.REMOVE_SORT, { currentModel }, 'click', 'desc');
+  consumer.send(RequestActions.REMOVE_SORT, { currentModel }, 'click');
+});
+
+test('REMOVE_SORT_WITH_PARAM', function(assert) {
+  assert.expect(2);
+
+  let parameters = { currency: 'USD' },
+      RevenueUSD = {
```

Comment:
Why capitalize first letter?

Reply:
it's all over so I didn't change it but not really necessary